python/ray/dag/tests/test_accelerated_dag.py

```diff
@@ -46,7 +47,15 @@ def append_to(self, lst):
     def inc_two(self, x, y):
         self.i += x
         self.i += y
-        return self.i
+        return x, y, self.i
+
+    def inc_four(self, a, b, x, y):
+        self.i += a
+        self.i += b
+        sum_a_b = a + b
+        self.j += x
+        self.j += y + sum_a_b
+        return a, b, x, y, self.i, self.j
```

Comment:
I don't think we need this extra logic, it's a bit confusing to figure out what output to expect. Probably fine to just return (a, b, x, y), since the thing we actually care about is just the order of inputs.